config/laratrust.php

```diff
@@ -46,7 +46,7 @@
         | NOTE: Currently the database check does not use cache.
         |
         */
-        'enabled' => env('LARATRUST_ENABLE_CACHE', true),
+        'enabled' => env('LARATRUST_ENABLE_CACHE', env('APP_ENV') === 'production'),
```

Comment:
Hmm I want to do more research here before this gets turned on (assuming this release enables that). Does all of the Laratrust stuff only get cached or is it backing up in the database as well? On shared hosting if it's only caching and never hitting the DB, that sounds like a disaster waiting to happen...


If this is still storing in the DB and then simply caching what's been put there, I see no need to disable this on dev environments

Reply:
This was a default change in the config file for laratrust. It is currently enabled (set to true) - this just gives you more flexibility in configuring which environments should be cached. Should be no change for us since this maintains the status quo on prod.

Reply:
Got it okay sounds good!